index.html

```diff
@@ -64,128 +65,7 @@ <h3 class="intro2">LET’S CONNECT</h3>
         </ul>
     </section>
     <section id="portfolio" class="project-section">
-        <div>
-            <div class="projects">
-                <div class="img-web">
-                    <img class="" src="images/project1.png" alt="project1">
-                </div>
-                <div class="project-description">
-                    <h1 class="title">Tonic</h1>
-                    <div class="role">
-                        <div class="role-name">CANOPY</div>
-                        <div class="role-dot"></div>
-                        <div class="role-job">Back End Dev</div>
-                        <div class="role-dot"></div>
-                        <div class="role-year">2015</div>
-                    </div>
-                    <div class="project-details">
-                        <p>
-                            A daily selection of privately personalized
-                            reads; no accounts or sign-ups required.
-                        </p>
-                    </div>
-                    <ul class="skills">
-                        <li class="skill set1">html</li>
-                        <li class="skill set2">css</li>
-                        <li class="skill set3">javascript</li>
-                    </ul>
-                    <div class="button-click">
-                        <button type="button" class="btn">See Project</button>
-                    </div>
-                </div>
-            </div>
-
-            <div class="projects">
-                <div class="img-web">
-                    <img class="" src="images/project2.png" alt="project1">
-                </div>
-                <div class="project-description">
-                    <h1 class="title">Multi-Post Stories</h1>
-                    <div class="role">
-                        <div class="role-name">CANOPY</div>
-                        <div class="role-dot"></div>
-                        <div class="role-job">Back End Dev</div>
-                        <div class="role-dot"></div>
-                        <div class="role-year">2015</div>
-                    </div>
-                    <div class="project-details">
-                        <p>
-                            A daily selection of privately personalized
-                            reads; no accounts or sign-ups required.
-                        </p>
-                    </div>
-                    <ul class="skills">
-                        <li class="skill set1">html</li>
-                        <li class="skill set2">css</li>
-                        <li class="skill set3">javascript</li>
-                    </ul>
-                    <div class="button-click">
-                        <button type="button" class="btn">See Project</button>
-                    </div>
-                </div>
-            </div>
-
-            <div class="projects">
```

Comment:
Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, margins, transparent background, width, and height). Also please implement the same on the mobile version.


Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/161938874-23267f6c-b9a6-4e0f-9bf0-3ef7a20f8d74.png) | ![image](https://user-images.githubusercontent.com/10332499/161938509-de707db2-da0b-4bf0-9a16-2f56882cf4da.png)

Reply:
- [ ] [REQUIRED] Just as the last reviewer recommended, kindly ensure that the popup properly follows the design for both desktop and mobile versions (e.g., font, colors, margins, transparent background, width, and height). 
Currently, you have vertical scroll bars you need to remove on the desktop popup, and the popup for the mobile version is not well aligned.
### Your design (Desktop)
![Screenshot from 2022-04-06 12-45-35](https://user-images.githubusercontent.com/58692590/161968817-e2425332-840d-4e21-840b-cb73b925fa2a.png)
![Screenshot from 2022-04-06 12-45-40](https://user-images.githubusercontent.com/58692590/161968858-e2867533-b120-42c9-8aa4-67dad0c1e038.png)
### Your design (Mobile)
![Screenshot from 2022-04-06 12-49-20](https://user-images.githubusercontent.com/58692590/161969241-51c10622-ec07-4242-ae85-e607de4dafb8.png)

Here is what your Popup is supposed to look like
### Figma Template
![Screenshot from 2022-04-06 13-10-20](https://user-images.githubusercontent.com/58692590/161971793-60308bfa-5dbb-4c04-b019-208576f33a1d.png)

Reply:
Hello, @Yorsyboy I understand your point. Please try to resize the sample popup from the template. You can see that it has its length longer than the width but your design has its width wider, consider imitating the design for the template ad all its fonts, font size, width, and height. I am sure if you adhere to this you will get the desired outcome. Well done 👍🏽 and Happy coding!!!